scripts/bash/buildInternalDependencies.sh

```diff
@@ -17,10 +17,6 @@ while [ "$1" != "" ]; do
         shift
         watch=true
         ;;
-    --withTypes)
```

Comment:
Currently we don't build TS types by default for JS packages that support them (eg `aggregator`). This is for historical reasons: when we first added this feature, TS was new to us and we didn't want to risk breaking existing functionality just to support it.

I feel that now is a good time to support TS types by default, but happy to re-consider

Reply:
I love this change! Having to manually rebuild the types each time after running `yarn` was starting to get tedious 😝